pysass.cpp

```diff
@@ -423,9 +423,14 @@ static Sass_Import_List _call_py_importer_f(
     PyObject* pyfunc = (PyObject*)sass_importer_get_cookie(cb);
     PyObject* py_result = NULL;
     Sass_Import_List sass_imports = NULL;
+    struct Sass_Import* previous;
+    const char* prev_path;
     Py_ssize_t i;
 
-    py_result = PyObject_CallFunction(pyfunc, PySass_IF_PY3("y", "s"), path);
+    previous = sass_compiler_get_last_import(comp);
+    prev_path = sass_import_get_abs_path(previous);
```

Comment:
do we need to free either of these strings?

Reply:
Oh, great question. I had meant to look into this and forgot. Will get back to you.

Reply:
nope, they don't need to be freed. The `sass_import_get_abs_path` returns a const char pointer to a member of the `Sass_Import` object, and `previous` is just a pointer from a `std::vector` member of the compiler. The pointers themselves are stack allocated, so they are freed when they go out of scope, and the compiler itself gets cleaned up in the calls to `sass_delete_file_context` and `sass_delete_data_context`.